shared/common/restful-authentication/token-method.mdx

```diff
@@ -4,4 +4,17 @@
 
     -   The Signaling token generated from your server
 
-    -   The Signaling user ID used to generate the Signaling token
\ No newline at end of file
+    -   The Signaling user ID used to generate the Signaling token
+
+- **AccessToken2**
+
+    You need to fill the `Authorization: agora token=` field with the RTM [AccessToken2](../develop/authentication) generated from your server.
+
+- **AccessToken**
+
+    You need to fill the `x-agora-token` and `x-agora-uid` fields in the request header with the following information:
+
+    - The RTM [AccessToken]((https://docs.agora.io/en/Real-time-Messaging/token_server_rtm?platform=All%20Platforms)) generated from your server
```

Comment:
Indenting - these two sections both belong to **Token authentications**.
<img width="918" alt="image" src="https://user-images.githubusercontent.com/101544891/192250495-1c65ccfd-800c-4831-81b1-686978eb21bb.png">

Reply:
Done